distributed/client.py

```diff
@@ -4055,6 +4055,21 @@ class will be instantiated with any extra keyword arguments.
 
         return self.sync(self._register_worker_plugin, plugin=plugin, name=name)
 
+    async def _unregister_worker_plugin(self, name):
+        responses = await self.scheduler.unregister_worker_plugin(name=name)
+
+        for response in responses.values():
+            if response["status"] == "error":
+                exc = response["exception"]
+                typ = type(exc)
+                tb = response["traceback"]
+                raise exc.with_traceback(tb)
+        return responses
+
+    def unregister_worker_plugin(self, name):
+        """Unregisters a lifecycle worker plugin"""
```

Comment:
Could you expand this docstring a little bit? You can check out the existing `Client.register_worker_plugin` docstring for an example of a similar docstring. In particular, it'd be good to mention that a plugin's `teardown` method is called when it's unregistered.

Reply:
I expanded this docstring, not sure if it's enough let me know if something is unclear.